### DISCUSSION TITLE
Upgrade sphinx-book-theme for Sphinx-4.x

### DISCUSSION THREAD

**Discussion by Vineet-Sharma29** [executablebooks/meta#447] [Q&A]
While installing `sphinx-book-theme`, I observed it auto-uninstalls `sphinx-4.x` and installs `sphinx-3.5.4`, since the theme is compatible with the latter version at present. Is the team looking to upgrade the theme ?

**Comment by choldgraf** [comment]
let's see if this works here: https://github.com/executablebooks/sphinx-book-theme/pull/364
